### DISCUSSION TITLE
PR created under an incorrect pattern

### DISCUSSION THREAD

**Discussion by JavierSegoviaCordoba** [renovatebot/renovate#21998] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

I have those two renovate files:

- https://github.com/JavierSegoviaCordoba/hubdle/blob/main/.github/renovate.json
- https://github.com/JavierSegoviaCordoba/reusable-workflows/blob/main/.github/renovate-reusable.json

The relevant part is:

```json
...
"packageRules": [
    {
      "matchPackagePatterns": [
        "androidx.compose.compiler:compiler",
        "org.jetbrains.kotlin:*"
      ],
      "groupName": "Compose Compiler and Kotlin"
    }
  ],
...
```

I would expect that any artifact under org.jetbrains.kotlinx: (check the pattern is without the latest x) would not fall under that pattern, but it is, as you can see on this PR:

https://github.com/JavierSegoviaCordoba/hubdle/pull/569

In which, the artifacts to be updated are:

- `org.jetbrains.kotlinx:kotlinx-coroutines-test`
- `org.jetbrains.kotlinx:kotlinx-coroutines-core`
- `org.jetbrains.kotlinx:kotlinx-coroutines-android`

Am I doing something wrong? If I am not doing something wrong, I can try to create a minimal repro.

### What you tried so far.

Nothing yet

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment]
It's a Regex pattern, and `kotlinx` matches `kotlin:*` because * means zero or more occurrences

**Reply by viceice** [reply]
use `:.+` at end as pattern

**Reply by rarkins** [reply, marked as answer]
Isn't dropping the * enough?

**Reply by JavierSegoviaCordoba** [reply]
It works, thank you!
